README.md

```diff
@@ -118,6 +119,49 @@ goldrush run reads=long_reads G=3e9
 
 **For more information about the GoldRush algorithm and tips for running GoldRush see our [wiki](https://github.com/bcgsc/goldrush/wiki)** 
 
+# System Requirements
+
+## Hardware Requirements
+
+GoldRush requires a standard computer with enough RAM to support the operations defined by the user. For the assembly of a human genome with a ~60X coverage, GoldRush requires a computer with:
+
+RAM: 64+ GB 
+
+and recommends a computer with at least:
+
+CPU: 24+ cores (48+ threads).
+
+The runtimes and RAM usage below are generated using a server-class system (144 Intel(R) Xeon(R) Gold 6254 CPU @ 3.1 GHz with 2.9 TB RAM) with 48 threads specified.
+
+| Data        | Time (h)    | RAM (GB)    |

```

Comment:
Since H. sapiens is repeated, could instead include that above the table, and then have a heading "Fold Coverage"

Reply:
yeah ill change that